deployment/stacks/service/base/template/service-search/service-account.yaml

```diff
@@ -3,4 +3,4 @@ kind: ServiceAccount
 metadata:
   name: uec-dos-api-sfs-service-account
   annotations:
-    eks.amazonaws.com/role-arn: arn:aws:iam::AWS_ACCOUNT_ID_TO_REPLACE:role/uec-dos-api-sfs-pod
+    eks.amazonaws.com/role-arn: arn:aws:iam::AWS_ACCOUNT_ID_TO_REPLACE:role/SERVICE_TAG_TO_REPLACE-PROFILE_TO_REPLACE-PROJECT_TAG_TO_REPLACE-dynamodb-access
```

Comment:
SFS may need more than just access to Dynamo in future.  Good idea having the PROFILE in the name though. Can we please use:
SERVICE_TAG_TO_REPLACE-PROFILE_TO_REPLACE-SA